<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchWidget</class>
 <widget class="QWidget" name="SearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="searchText"/>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="searchList">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="indentation">
      <number>0</number>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="headerDefaultSectionSize">
      <number>200</number>
     </attribute>
     <attribute name="headerShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>200</number>
     </attribute>
     <attribute name="headerShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>searchText</sender>
   <signal>returnPressed()</signal>
   <receiver>SearchWidget</receiver>
   <slot>emitStartSearch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchButton</sender>
   <signal>clicked()</signal>
   <receiver>SearchWidget</receiver>
   <slot>emitStartSearch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>427</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchText</sender>
   <signal>textChanged(QString)</signal>
   <receiver>SearchWidget</receiver>
   <slot>searchTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>startSearch(QString,MatchType)</signal>
  <slot>emitStartSearch()</slot>
  <slot>searchTextChanged(QString)</slot>
 </slots>
</ui>
